src/Microsoft.ML.StandardTrainers/Standard/Online/AveragedPerceptron.cs

```diff
@@ -82,6 +82,11 @@ public sealed class AveragedPerceptronTrainer : AveragedLinearTrainer<BinaryPred
         /// </summary>
         public sealed class Options : AveragedLinearOptions
         {
+            public Options()
+            {
+                NumberOfIterations = 10;
```

Comment:
I see this correctly changed the number of iterations in the MAML based unit tests:
```diff
-Warning: Skipped 15 instances with missing features during training (over 1 iterations; 15 inst/iter)
+Warning: Skipped 150 instances with missing features during training (over 10 iterations; 15 inst/iter)
```

Do we have a unit test for `AveragedPerceptron` using the Estimator API? If you haven't, it would be good to verify the new defaults take hold for the AP Estimator API.

Reply:
We do. The `OvaAveragedPerceptron` test in `OvaTests.cs` is an Estimator API test. I have confirmed using that test that the new defaults are there correctly as well.

Reply:
SGTM